cmd/kpod/create.go

```
@@ -0,0 +1,899 @@
+	}
+
+	// RESOURCES - CPU
+
+	if config.resources.cpuShares != 0 {
+		spec.Linux.Resources.CPU.Shares = &config.resources.cpuShares
+	}
+	if config.resources.cpuQuota != 0 {
+		spec.Linux.Resources.CPU.Quota = &config.resources.cpuQuota
+	}
+	if config.resources.cpuPeriod != 0 {
+		spec.Linux.Resources.CPU.Period = &config.resources.cpuPeriod
+	}
+	if config.resources.cpuRtRuntime != 0 {
+		spec.Linux.Resources.CPU.RealtimeRuntime = &config.resources.cpuRtRuntime
+	}
+	if config.resources.cpuRtPeriod != 0 {
+		spec.Linux.Resources.CPU.RealtimePeriod = &config.resources.cpuRtPeriod
+	}
+	if config.resources.cpus != "" {
+		spec.Linux.Resources.CPU.Cpus = config.resources.cpus
+	}
+	if config.resources.cpusetMems != "" {
+		spec.Linux.Resources.CPU.Mems = config.resources.cpusetMems
+	}
+
+	// RESOURCES - PIDS
+	if config.resources.pidsLimit != 0 {
+		spec.Linux.Resources.Pids.Limit = config.resources.pidsLimit
+	}
+
+	/*
+				Capabilities: &spec.LinuxCapabilities{
+				// Rlimits []PosixRlimit // Where does this come from
+				// Type string
+				// Hard uint64
+				// Limit uint64
+				// NoNewPrivileges bool // No user input for this
+				// ApparmorProfile string // No user input for this
+				OOMScoreAdj: &config.resources.oomScoreAdj,
+				// Selinuxlabel
+			},
+			Hooks: &spec.Hooks{},
+			//Annotations
+				Resources: &spec.LinuxResources{
+					Devices: config.GetDefaultDevices(),
+					BlockIO: &blkio,
+					//HugepageLimits:
+					Network: &spec.LinuxNetwork{
+					// ClassID *uint32
+					// Priorites []LinuxInterfacePriority
+					},
+				},
+				//CgroupsPath:
+				//Namespaces: []LinuxNamespace
+				//Devices
+				Seccomp: &spec.LinuxSeccomp{
+				// DefaultAction:
+				// Architectures
+				// Syscalls:
+				},
+				// RootfsPropagation
+				// MaskedPaths
+				// ReadonlyPaths:
+				// MountLabel
+				// IntelRdt
+			},
+		}
+	*/
+	return &spec, nil
+}
+
+func getStatFromPath(path string) unix.Stat_t {
```

Doesn't this ignore errors? Should we be ignoring errors?